src/hotspot/share/gc/shenandoah/shenandoahHeap.cpp

```diff
@@ -623,6 +623,11 @@ size_t ShenandoahHeap::used() const {
   return Atomic::load_acquire(&_used);
 }
 
+size_t ShenandoahHeap::live() const {
+  size_t live = Atomic::load_acquire(&_live);
```

Comment:
I understand you copy-pasted from the same file. We have removed `_acquire` with #2504. Do `Atomic::load` here.

Reply:
...which also means you want to merge from master to get recent changes?

Reply:
Yep. Done.